src/app/external-cluster-wizard/steps/summary/external-cluster-details/external-cluster-details-aks/component.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Component, Input, ViewEncapsulation, OnInit} from '@angular/core';
+import {AKSClusterSpec, AKSCloudSpec} from '@shared/entity/provider/aks';
+import {ExternalClusterModel} from '@shared/entity/external-cluster';
+
+@Component({
+  selector: 'km-external-cluster-details-aks',
+  templateUrl: './template.html',
+  encapsulation: ViewEncapsulation.None,
+})
+export class ExternalClusterDetailsAKSComponent implements OnInit {
+  @Input() cluster: ExternalClusterModel;
+
+  aks: AKSCloudSpec;
+  spec: AKSClusterSpec;
+
+  ngOnInit(): void {
```

Comment:
Since `cluster` value can change from parent so should this be `ngOnChanges` instead?

Reply:
It won't change, as this is a static component.

Reply:
Yes, but this component has no control over when the `cluster` value changes in parent and when it changes then this component won't display latest values.

Reply:
this is why I'm only rendering this component when the user is on the last step, it is destroyed as soon as the user goes back.

Reply:
This component should be independent of the parent component's logic so it works fine if we try to reuse it somewhere else.